### PR TITLE
feat: bump versions

### PR DESCRIPTION
**Pull request by peterdeme**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,13 +11,13 @@
         }
     ],
     "require": {
-        "php": ">=7.2.0",
+        "php": ">=7.3",
```
Comment (ferhatelmas) [inline composer.json:14]: can we have 7.4 since we don't have it 7.3 in ci?

Comment (peterdeme) [reply composer.json:14]: Yeah but I wanted to copy this: https://github.com/GetStream/stream-php/blob/main/composer.json#L14

Although in the CI, I was thinking about testing with the latest 7.* and latest 8.*

Comment (ferhatelmas) [reply composer.json:14]: so let's add this too https://github.com/GetStream/stream-php/blob/main/.github/workflows/ci.yml#L11

Comment (peterdeme) [reply composer.json:14]: ✅

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Pull request merged**